lib/core-server/src/core-presets.test.ts

```diff
@@ -9,9 +9,13 @@ import { executor as managerExecutor } from './manager/builder';
 import { buildDevStandalone } from './build-dev';
 import { buildStaticStandalone } from './build-static';
 
+// nx-ignore-next-line
```

Comment:
These were some imports that created circular dependencies. These comments ignore the dependencies generated by these lines.